src/api/java/com/minecolonies/api/colony/requestsystem/location/ILocation.java

```diff
@@ -0,0 +1,34 @@
+package com.minecolonies.api.colony.requestsystem.location;
+
+import net.minecraft.util.math.BlockPos;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Interface used to describe locations in the world.
+ */
+public interface ILocation
```

Comment:
I am not sure this class Needs to be here yet. It is not part of the core request System, but more of a Thing like IToken.